[2024-06-25T08:40:45.124+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-25T08:40:45.185+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dbt_snowflake_v5.dbt_run scheduled__2024-06-24T00:00:00+00:00 [queued]>
[2024-06-25T08:40:45.200+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dbt_snowflake_v5.dbt_run scheduled__2024-06-24T00:00:00+00:00 [queued]>
[2024-06-25T08:40:45.201+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-25T08:40:45.228+0000] {taskinstance.py:2330} INFO - Executing <Task(DockerOperator): dbt_run> on 2024-06-24 00:00:00+00:00
[2024-06-25T08:40:45.242+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dbt_snowflake_v5', 'dbt_run', 'scheduled__2024-06-24T00:00:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp3nt31jdg']
[2024-06-25T08:40:45.246+0000] {standard_task_runner.py:91} INFO - Job 9: Subtask dbt_run
[2024-06-25T08:40:45.265+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1002) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-25T08:40:45.267+0000] {standard_task_runner.py:63} INFO - Started process 1003 to run task
[2024-06-25T08:40:45.335+0000] {task_command.py:426} INFO - Running <TaskInstance: dbt_snowflake_v5.dbt_run scheduled__2024-06-24T00:00:00+00:00 [running]> on host 40877dbc4b09
[2024-06-25T08:40:45.495+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dbt_snowflake_v5' AIRFLOW_CTX_TASK_ID='dbt_run' AIRFLOW_CTX_EXECUTION_DATE='2024-06-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-24T00:00:00+00:00'
[2024-06-25T08:40:45.496+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-25T08:40:45.559+0000] {docker.py:366} INFO - Starting docker container from image ghcr.io/dbt-labs/dbt-snowflake:latest
[2024-06-25T08:40:45.565+0000] {docker.py:374} WARNING - Using remote engine or docker-in-docker and mounting temporary volume from host is not supported. Falling back to `mount_tmp_dir=False` mode. You can set `mount_tmp_dir` parameter to False to disable mounting and remove the warning
[2024-06-25T08:40:48.849+0000] {docker.py:436} INFO - [0m08:40:48  Running with dbt=1.8.2
[2024-06-25T08:40:49.645+0000] {docker.py:436} INFO - [0m08:40:49  Registered adapter: snowflake=1.8.3
[2024-06-25T08:40:50.551+0000] {docker.py:436} INFO - [0m08:40:50  Found 5 models, 9 data tests, 2 sources, 561 macros
[2024-06-25T08:40:50.556+0000] {docker.py:436} INFO - [0m08:40:50
[2024-06-25T08:40:52.688+0000] {docker.py:436} INFO - [0m08:40:52  Concurrency: 1 threads (target='dev')
[2024-06-25T08:40:52.689+0000] {docker.py:436} INFO - [0m08:40:52
[2024-06-25T08:40:52.706+0000] {docker.py:436} INFO - [0m08:40:52  1 of 5 START sql view model dbt_schema.stg_tpch_line_items ..................... [RUN]
[2024-06-25T08:40:54.140+0000] {docker.py:436} INFO - [0m08:40:54  1 of 5 OK created sql view model dbt_schema.stg_tpch_line_items ................ [[32mSUCCESS 1[0m in 1.43s]
[2024-06-25T08:40:54.143+0000] {docker.py:436} INFO - [0m08:40:54  2 of 5 START sql view model dbt_schema.stg_tpch_orders ......................... [RUN]
[2024-06-25T08:40:54.979+0000] {docker.py:436} INFO - [0m08:40:54  2 of 5 OK created sql view model dbt_schema.stg_tpch_orders .................... [[32mSUCCESS 1[0m in 0.83s]
[2024-06-25T08:40:54.987+0000] {docker.py:436} INFO - [0m08:40:54  3 of 5 START sql table model dbt_schema.int_order_items ........................ [RUN]
[2024-06-25T08:41:06.794+0000] {docker.py:436} INFO - [0m08:41:06  3 of 5 OK created sql table model dbt_schema.int_order_items ................... [[32mSUCCESS 1[0m in 11.80s]
[2024-06-25T08:41:06.800+0000] {docker.py:436} INFO - [0m08:41:06  4 of 5 START sql table model dbt_schema.int_order_items_summary ................ [RUN]
[2024-06-25T08:41:08.502+0000] {docker.py:436} INFO - [0m08:41:08  4 of 5 OK created sql table model dbt_schema.int_order_items_summary ........... [[32mSUCCESS 1[0m in 1.70s]
[2024-06-25T08:41:08.506+0000] {docker.py:436} INFO - [0m08:41:08  5 of 5 START sql table model dbt_schema.fct_orders ............................. [RUN]
[2024-06-25T08:41:10.395+0000] {docker.py:436} INFO - [0m08:41:10  5 of 5 OK created sql table model dbt_schema.fct_orders ........................ [[32mSUCCESS 1[0m in 1.89s]
[2024-06-25T08:41:10.400+0000] {docker.py:436} INFO - [0m08:41:10
[2024-06-25T08:41:10.401+0000] {docker.py:436} INFO - [0m08:41:10  Finished running 2 view models, 3 table models in 0 hours 0 minutes and 19.84 seconds (19.84s).
[2024-06-25T08:41:10.478+0000] {docker.py:436} INFO - [0m08:41:10
[2024-06-25T08:41:10.479+0000] {docker.py:436} INFO - [0m08:41:10  [32mCompleted successfully[0m
[2024-06-25T08:41:10.480+0000] {docker.py:436} INFO - [0m08:41:10
[2024-06-25T08:41:10.481+0000] {docker.py:436} INFO - [0m08:41:10  Done. PASS=5 WARN=0 ERROR=0 SKIP=0 TOTAL=5
[2024-06-25T08:41:12.732+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-25T08:41:12.772+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dbt_snowflake_v5, task_id=dbt_run, run_id=scheduled__2024-06-24T00:00:00+00:00, execution_date=20240624T000000, start_date=20240625T084045, end_date=20240625T084112
[2024-06-25T08:41:12.811+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-25T08:41:12.846+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-25T08:41:12.849+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
